src/hotspot/share/runtime/arguments.hpp

```diff
@@ -462,9 +462,8 @@ class Arguments : AllStatic {
   static int  PropertyList_readable_count(SystemProperty* pl);
 
   static bool is_internal_module_property(const char* option);
-  static bool is_add_modules_property(const char* key);
-  static unsigned int addmods_count() { return  _addmods_count; }
-  static bool is_module_path_property(const char* key);
+  static bool is_non_cds_compatible_internal_module_property(const char* property);
```

Comment:
Suggestion on the function name:
    `is_incompatible_cds_internal_module_property`